### DISCUSSION TITLE
DPA2  dpgen

### DISCUSSION THREAD

**Discussion by Cc-12342234** [deepmodeling/dpgen#1668] [Q&A]
Dear Developer,

I would like to ask if dpgen supports training with the dpa2 version 300b4? I have not been able to find relevant reference materials on this. Could you kindly provide some advice or guidance on this matter?

Thank you!

**Comment by njzjz** [comment]
It has been supported in the devel branch, via [`train_backend`](https://docs.deepmodeling.com/projects/dpgen/en/devel/run/param.html#argument:run_jdata[mlp_engine=dp]/train_backend).

We will release the new version in this week.
